A6.md

```diff
@@ -0,0 +1,398 @@
+gRPC Retry Design
+----
+* Author(s): [Noah Eisen](https://github.com/ncteisen) and [Eric Gribkoff](https://github.com/ericgribkoff)
+* Approver: a11r
+* Status: Draft
+* Implemented in: All Languages
```

Comment:
I think this line is intended to indicate where this proposal is *already* implemented.  Given that we haven't yet implemented this in any language, I think this field should be blank.